packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
+const GLOB_PATTERNS = '**\/*.{html,js,css}';
```

Comment:
Yes!
It'd be great if `sw-build` can have a set of defaults, that way every plugin does not have to define them